src/Organizations/OrganizationDetails/OrganizationDetails.js

```diff
@@ -310,20 +324,33 @@ const OrganizationDetails = ({
             </Accordion>
 
             <Accordion
-              id={ORGANIZATION_SECTIONS.interfacesSection}
-              label={ORGANIZATION_SECTION_LABELS[ORGANIZATION_SECTIONS.interfacesSection]}
+              id={interfacesSection}
+              label={ORGANIZATION_SECTION_LABELS[interfacesSection]}
             >
               <OrganizationInterfacesContainer
                 interfaceIds={organization.interfaces}
               />
             </Accordion>
 
+            {
+              Boolean(organization.isVendor && organization.isDonor) && (
+              <Accordion
+                id={donorContacts}
+                label={ORGANIZATION_SECTION_LABELS[donorContacts]}
+              >
+                {/*
+                  TODO: add donor contacts list component https://issues.folio.org/browse/UIORGS-394
+                */}
+              </Accordion>
```

Comment:
* `Boolean(foo && bar)` seems superfluous to me since `foo && bar` will already evaluate to `boolean` but 🤷 
* I am not a fan of placeholder stories that insert a broken/empty UI element. Essentially, they add a bug ("accordion is present but has no content"). I guess that's an issue to take up with your PO, but I would strongly prefer UIORGS-398 to be merged after UIORGS-394 was complete.

Reply:
Sure. Thank you for your comments.